controllers/medusa/controllers_test.go

```diff
@@ -173,6 +181,7 @@ func setupMedusaRestoreJobTestEnv(t *testing.T, ctx context.Context) *testutils.
 				Scheme:           scheme.Scheme,
 				ClientFactory:    medusaRestoreClientFactory,
 			}).SetupWithManager(dataPlaneMgr)
+			secretswebhook.SetupSecretsInjectorWebhook(dataPlaneMgr)
```

Comment:
Question: I'm not sure what the changes in this file are doing? They don't look related to the purpose of this PR...

Reply:
It's necessary because this envtest now needs to create cassandra pods and get some mock interactions, whereas before it was using the standalone pod which we were faking.
We can't create pods in the envtest if we don't have the webhook present.

Reply:
Because they need secrets mounted I presume and those secrets come from the webhook? Roger, sounds fine to me.